index.html

```diff
@@ -5,37 +5,94 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Ben's Restaurant</title>
-    <link rel="stylesheet" href="/style/css/main.css">
+    <link rel="stylesheet" href="style/css/main.css">
 </head>
 <body>
     <header>
         <div class="menu-btn">
             <span class="burger"></span>
         </div>
         <nav class="navbar">
+            <a href="index.html" class="logo"><img src="assets/1280px-Daum_Kakao_wordmark_-_2014.svg.png" alt="logo_img"></a>
             <ul class="menu-navbar">
                 <li class="menu-navbar__item">
-                    <a href="/webpages/about.html">about</a>
```

Comment:
- [ ] It will be better 😊 if you put the navigation links in a suitable position on the mobile version for example you can put them at the center to make it nice for the user 💯
- [ ] Also consider increasing the size of the link to be great for the users and to be like the Behnace design 😃
- [ ] Also consider decreasing the margin between the ```X``` icon and the links to be nice for the users 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
 | --- |
|![image](https://user-images.githubusercontent.com/83360636/233070233-6f871d18-da8e-45c4-9821-63e0c3e69356.png)   |